src/debugger/nodeDebugWrapper.ts

```diff
@@ -135,8 +136,16 @@ Telemetry.init("react-native-debug-adapter", version, true).then(() => {
             args.target || "simulator"
         ];
 
+        if (!isNullOrUndefined(args.logCatArguments)) { // We add the parameter if it's defined (adapter crashes otherwise)
+            args.args = args.args.concat([args.logCatArguments]);

```

Comment:
What kind of arguments are valid logCatArguments? Looking at the docs, logcat takes a bunch of things potentially, so I might expect to provide an array of arguments to pass to it and that won't work with the way things are set up here.